Classes/Issues/IssueManagingContextController.swift

```diff
@@ -323,10 +335,22 @@ final class IssueManagingContextController: NSObject, ContextMenuDelegate {
 
         let selected = controller.selected
         guard controller.selectionChanged(newValues: selected) else { return }
+
+        if controller.wasDismissedByDone {
```

Comment:
Can we put this logic in a function, as it seems to be repeated from lines 312-320?